test/check-machines-create

```diff
@@ -1478,8 +1478,11 @@ class TestMachinesCreate(VirtualMachinesCase):
 
         def createThenInstallTest(self, dialog, installFromVmDetails=False, tryWithFailInstall=False):
             self._tryCreateThenInstall(dialog, installFromVmDetails, tryWithFailInstall) \
-                ._assertScriptFinished() \
-                ._assertDomainDefined(dialog.name, dialog.connection) \
+                ._assertScriptFinished()
+
+            wait(lambda: "shut off" if tryWithFailInstall else "running" in self.machine.execute(f"virsh dominfo --domain {dialog.name} | grep State"), delay=3)
```

Comment:
Note, in my local test runs it was always shut off here, as `_tryCreateThenInstall()` runs `forceOff`. (Before that, it's running)